use-case-shopping/README.md

```diff
@@ -0,0 +1,93 @@
+<!-- Copyright 2019 Oath Inc. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root. -->
+
+# Vespa sample applications - e-commerce
+
+A sample application showcasing a simple e-commerce site built with Vespa. 
+Please refer to [Vespa use cases](https://docs.vespa.ai/documentation/use_cases.html) 
+for more information.
+
+Included scripts to convert data from Julian McAuley's Amazon product data set
+(http://jmcauley.ucsd.edu/data/amazon/links.html)
+to a Vespa data feed. This repository contains a small sample of this data from 
+the sports and outdoor category, but you can download other data from the site 
+above and use the scripts to convert. 
+
+### How to run
```

Comment:
Add what is required to run the sampleapp, like python 3

Reply:
or, we could pack python3 with the image and run the processing inside the container